### ISSUE TITLE
phantomjs is missing for linux?

### ISSUE THREAD

**Issue by ceball** [AnacondaRecipes/aggregate#38]
Seems like there are osx-64, win-32, and win-64 packages, but none for linux.

https://anaconda.org/conda-forge/phantomjs
https://github.com/conda-forge/phantomjs-feedstock

**Comment by jakirkham** [comment]
Should this have been filed at conda-forge?

In any event, am seeing osx-64, linux-64, win-64, and win-32 in descending order (screenshot below). Is there something else that prompted this issue (e.g. trouble installing on Linux)?

<img width="1552" alt="screen shot 2018-04-27 at 03 33 44" src="https://user-images.githubusercontent.com/3019665/39350320-d1893e12-49cb-11e8-9cda-ae9ce94c429e.png">

**Comment by msarahan** [comment]
Yes, issues with conda-forge packages should be filed on the corresponding conda-forge feedstock repo.

**Comment by ceball** [comment]
Ah, I see why I have caused confusion. As requested, I pasted links to the relevant package on conda-forge, but of course it looks like I'm suggesting the package is missing from conda-forge. Sorry about that.

**Comment by msarahan** [comment]
I believe bokeh ships their own phantomjs, which is probably repackaged from upstream binaries.  A very long time ago, I attempted to package phantomjs, but I only remember it was a huge pain, and I may not have succeeded.  We can put it on our list to revisit.
